src/arena/CreateSprintTemplate.js

```diff
@@ -143,19 +146,108 @@ function CreateSprintTemplate(props) {
     getSanityItems();
   }, []);
 
+  useEffect(() => {
+    getConfiguration();
+  }, []);
+
+  useEffect(() => {
+    handleChange();
+  }, [title, columns, handleChange]);
+
+  // pulls the /templates api and sets the missions to the templates
+  async function getConfiguration() {
+    let options = await RestAPI.get("pareto", "/templates");
+    setMissions(options.map((option) => option.title));
+  }
+  const minimumOptions =
```

Comment:
Nit: (doesn't need any action) It might be easier for a totally new contributor (who's never looked at the file before) if this were named "hasMinimumOptionsAdded" or "meetsMinimumOptionsThreshold" ...for someone seeing code for the first time, it's helpful to name variables in a way that lets the reader know to expect a yes or no (true or false) answer vs a number (I could see reading minimumOptions later in the code and thinking it would be a number not a true/false.)